tensorflow-framework/src/main/java/org/tensorflow/framework/activations/Activation.java

```diff
@@ -41,28 +36,29 @@ protected Activation(Ops tf) {
   }
 
   /**
-   * Sets the TensorFlow Ops
+   * Gets the TensorFlow Ops
    *
-   * @param tf the TensorFlow Ops
+   * @return the TensorFlow Ops
    */
-  protected void setTF(Ops tf) {
-    this.tf = tf;
+  protected Ops getTF() {
+    return this.tf;
   }
 
   /**
-   * Gets the TensorFlow Ops
+   * Sets the TensorFlow Ops
    *
-   * @return the TensorFlow Ops
+   * @param tf the TensorFlow Ops
    */
-  protected Ops getTF() {
-    return this.tf;
+  protected void setTF(Ops tf) {
+    this.tf = tf;
   }
 
   /**
    * Gets the calculation operation for the activation.
    *
    * @param input the input tensor
    * @return The operand for the activation
+   * @param <T> the data type of the input and result
```

Comment:
can you [reorder](https://google.github.io/styleguide/javaguide.html#s7.1.3-javadoc-block-tags) these tags so that the `@param` are all before `@return`? I think the generic parameter should be the first one, I'm not sure if there is a standard for this. Or, if you prefer, I would also be totally comfortable not documenting at all the generic parameter of these method, which is quite explicit. Your choice.

This comment applies to other places in that PR.

Reply:
I did find this document from Oracle, [How to Write Doc Comments for the Javadoc Tool](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html).
`@param` should be before `@return`, but it doesn't mention anything about ordering the generic within the params.
It just says `Multiple @param tags should be listed in argument-declaration order. This makes it easier to visually match the list to the declaration.`

I will fix by moving the `@param<T>` to before the `@return`

Reply:
Javadoc will warn on any missing parameters, generic or otherwise, so I think it's better to have all parameters documented.